openapi.yaml

```diff
@@ -16985,6 +17032,346 @@ paths:
         source: >
           curl -H "Authorization: Bearer $TOKEN" \
             https://api.linode.com/v4/object-storage/transfer/
+  /placement/groups:
+    x-linode-cli-command: placement
+    get:
+      parameters:
+        - $ref: '#/components/parameters/pageOffset'
+        - $ref: '#/components/parameters/pageSize'
+      summary: Placement Groups List
+      description: >
+        Returns a paginated list of placement groups you have permission to view.
+      tags:
+        - Placement Groups
+      operationId: getPlacementGroups
+      x-linode-cli-action:
+        - list
+        - ls
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - placement:read_only
+      responses:
+        '200':
+          description: |
+            Returns an array of all placement groups on your Account.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/PlacementGroup'
+                  page:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/page'
+                  pages:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/pages'
+                  results:
+                    $ref: '#/components/schemas/PaginationEnvelope/properties/results'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4/placement/groups
+      - lang: CLI
+        source: >
+          linode-cli placement-list
+    post:
+      x-linode charge: true
+      x-linode grant: add_placement
+      summary: Placement Group Create
+      description: |
+        Creates a placement group on your account. Placement groups let you set the physical location of your compute instances in a data center (region) to best suit your need.
+
+        * For this request to complete successfully, your user must have the `add_placement` grant.
+        * We offer an [example API workflow](/docs/products/compute/compute-instances/guides/placement-groups/#create-a-placement-group) you can follow to create a placement group.
+
+        **Note**: With the beta release, this service is only available in [select regions](/docs/products/compute/compute-instances/guides/placement-groups/#availability).
+      tags:
+      - Placement Groups
+      operationId: createPlacementGroup
+      x-linode-cli-action: create
```

Comment:
Just to future proof ourselves for any more placement-related objects, could this (and the other CRUD commands) follow the format `group-{action}`? (e.g. `group-create`)

Reply:
All of the other operations in the openapi.yaml file follow an `operationId` convention of:

<action><ObjectName> (`getThisObject`). So, I'm just trying to stay consistent.

Reply:
@Vernholio The operationId can stay as-is from my perspective 👍
 
In this case I think I was referring to updating `x-linode-cli-action`.

e.g. 
```
x-linode-cli-action: group-create
```

Reply:
Aha! Sorry for not seeing this. Of course you're talking about the CLI action, not the `operationId` (It IS what I asked you to look at, isn't it? )  <SMH>

Reply:
No problem at all!